SMCalloutView.m

```diff
@@ -38,6 +38,7 @@ @interface SMCalloutView ()
 @property (nonatomic, strong) UILabel *titleLabel, *subtitleLabel;
 @property (nonatomic, assign) SMCalloutArrowDirection currentArrowDirection;
 @property (nonatomic, assign) BOOL popupCancelled;
+@property (nonatomic, assign) BOOL useCustomContentViewMargin;
```

Comment:
This should be eliminated entirely, assuming `contentViewInset` was set to a default value in the initializer.